docs/tutorial.md

```diff
@@ -204,7 +204,7 @@ kubectl create secret docker-registry regcred \
                     --docker-email=<your-email>
 ```
 
-To be able to use this `secret` the `TaskRun` needs to use a `ServiceAccount`.
+To be able to use this `secret`, the `TaskRun` needs to use a `ServiceAccount`.
```

Comment:
Remove "be able to" - these are empty words that add no meaning. Change to "To use this `secret`...".